src/ui/components/DropdownMenu/styles.scss

```diff
@@ -0,0 +1,88 @@
+@import "~core/css/inc/mixins";
+@import "~amo/css/inc/vars";
+@import "~ui/css/vars";
+
+.DropdownMenu {
+  @include text-align-start();
+
+  display: inline-block;
+  padding-bottom: 6px;
+  padding-top: 6px;
+  position: relative;
+}
+
+.DropdownMenu-text {
+  @include font-regular();
+
+  color: $white;
+  font-size: $font-size-s;
+  font-weight: normal;
+  text-decoration: none;
+
+  .Icon-inverted-caret {
+    @include margin-start(5px);
+
+    height: 10px;
+    opacity: 0.6;
+    transition: opacity $transition-short ease-in-out;
+    width: 9px;
+  }
+}
+
+.DropdownMenu--active {
+  .DropdownMenu-items {
+    display: block;
+  }
+
+  .Icon-inverted-caret {
+    opacity: 1;
+  }
+}
+
+// This is needed because `:hover` on mobile prevents the main button to close
+// the dropdown menu. On mobile, `.DropdownMenu--active` controls the
+// visibility of the menu. On "large" devices (and above), `:hover` also opens
+// the menu.
+@include respond-to(large) {
```

Comment:
Similarly: I prefer when the selectors are at the top and the `respond-to(large) {}` things are inside them. It makes it easier to reason about an element's style rather than a screen size's style, because some things are the same across sizes but others aren't. I think it's a better way to organise and would prefer we do:

```scss
.SomeClass {
  @include respond-to(large) {
    display: block;
  }
}
```

going forward. Sorry if I missed that in previous reviews. If it's missing from the CSS guides I should add it.